hangupsbot/plugins/slackrtm.py

```diff
@@ -1196,6 +1196,8 @@ def handle_reply(self, reply):
 
     @asyncio.coroutine
     def handle_ho_message(self, event):
+        if self.bot._user_list.get_user(event.user_id).is_self:
```

Comment:
A smaller access path is [`event.from_bot`](https://github.com/hangoutsbot/hangoutsbot/blob/master/hangupsbot/handlers.py#L123..L126) which stores the same boolean value.